checks/remotesettings/certificates_expiration.py

```diff
@@ -50,12 +56,15 @@ async def run(server: str, min_remaining_days: int) -> CheckResult:
 
     # Second, deduplicate the list of x5u URLs and fetch them in parallel.
     x5us = list(set(metadata["signature"]["x5u"] for metadata in results))
-    futures = [fetch_cert(x5u) for x5u in x5us]
+    futures = [fetch_certs(x5u) for x5u in x5us]
     results = await run_parallel(*futures)
```

Comment:
Does this work when `futures` is a list of lists of futures? I believe the type changed from `List<Future<_>>` to `List<List<Future<_>>>`. I'm not very familiar with Python's async tools though, so maybe this is fine?

Reply:
It's a bit simpler than that, `fetch_certs()` used to return `Future<Cert>` and now returns `Future<List<Cert>>`

Reply:
Ahh, right. So the type of `futures` is `List<Future<List<Cert>>>`, so `run_parallel(*futures)` still works fine. Got it.